lib/rqrcode/qrcode/qr_code.rb

```diff
@@ -18,6 +18,13 @@ module RQRCode #:nodoc:
     :mode_kanji         => 1 << 3
   }
 
+  QRMODE_NAME = {
+    :number        => :mode_number,
+    :alphanumeric  => :mode_alpha_numk,
+    :'8bit_byte'   => :mode_8bit_byte,
```

Comment:
Maybe just use :byte for the 8bit mode? Quoted symbols are kind of unpleasant to write.